apps/dapp-console/app/console/useDialogContent.tsx

```diff
@@ -170,6 +173,33 @@ const useDialogContent = () => {
     })
   }, [authenticated, login])
 
+  const privyContent = useMemo(() => {
+    return renderDialog({
+      ...privyMetadata,
+      primaryButton: !authenticated
+        ? loginButton('Sign in to apply', privyMetadata.label)
+        : privyMetadata.primaryButton,
+    })
+  }, [authenticated, login])
+
+  const bwareContent = useMemo(() => {
+    return renderDialog({
+      ...bwareMetadata,
+      primaryButton: !authenticated
+        ? loginButton('Sign in to apply', bwareMetadata.label)
+        : bwareMetadata.primaryButton,
+    })
+  }, [authenticated, login])
+
+  const sherlockContent = useMemo(() => {
+    return renderDialog({
+      ...sherlockMetadata,
+      primaryButton: !authenticated
+        ? loginButton('Sign in to apply', sherlockMetadata.label)
+        : sherlockMetadata.primaryButton,
+    })
+  }, [authenticated, login])
```

Comment:
nit: might be worth creating a helper function/hook since this pattern is repeated